.goreleaser.yml

```
@@ -31,7 +31,8 @@ builds:
       goarch: '386'
   binary: '{{ .ProjectName }}_v{{ .Version }}'
 archives:
-- format: zip
+- formats:
+    - zip
```

Sorry dumb question. Is this indented too much?
```suggestion
  - zip
```

---

YAML supports either and the rest of this file is this way so kept it consistent.

---

Sounds good!